app/controllers/users/passwords_controller.rb

```diff
@@ -9,7 +9,8 @@ class PasswordsController < Devise::PasswordsController
       # recover passwords. However, we always want to return success.
       flash[:success] = t('upaya.notices.password_reset')
       redirect_to after_sending_reset_password_instructions_path_for(
-        resource_name)
+        resource_name
+      )
```

Comment:
Yes, but I'm planning on updating this method in a separate PR to reduce Devise lookups. For example, we can replace `resource_class` with `User`, and `resource_name` with `user`, which will shorten everything and make it fit nicely within 80 characters.